examples/web-sdk-sample/README.md

```diff
@@ -122,4 +122,40 @@ Record data sent and received during WebSocket communication:
 
 ```
 
+### Upload files to S3 bucket
```

Comment:
use `Amazon S3` instead of `S3` in document

Reply:
fixed